### PR TITLE
Add link to SEMAPV docs

### PR DESCRIPTION
**Pull request by joeflack4**

### CODE REVIEW

File: src/sssom/constants.py [hunk 1]
```diff
@@ -181,7 +181,10 @@
 
 
 class SEMAPV(Enum):
-    """SEMAPV Enum containing different mapping_justification."""
+    """SEMAPV Enum containing different mapping_justification.
+    
+    See also: https://mapping-commons.github.io/semantic-mapping-vocabulary/#matchingprocess
```
Comment (joeflack4) [inline src/sssom/constants.py:186]: One-liner

**Review by matentzn — DISMISSED** [review]
LGTM but QC errors need fixing.

**Comment by joeflack4** [comment]
I made this change in GitHub. I'll have to load this up in IDE / terminal / use black / amend but I can do that soon.

**Review by matentzn — APPROVED** [review]

**Comment by joeflack4** [comment]
Looks like you did that. Thanks!

**Pull request merged**